src/execution/compiler/codegen.cpp

```diff
@@ -584,11 +584,10 @@ ast::Expr *CodeGen::TableIterClose(ast::Expr *table_iter) {
   return call;
 }
 
-ast::Expr *CodeGen::IterateTableParallel(catalog::table_oid_t table_oid, ast::Identifier col_oids,
-                                         ast::Expr *query_state, ast::Expr *exec_ctx, ast::Identifier worker_name) {
-  ast::Expr *call = CallBuiltin(
-      ast::Builtin::TableIterParallel,
-      {Const32(table_oid.UnderlyingValue()), MakeExpr(col_oids), query_state, exec_ctx, MakeExpr(worker_name)});
+ast::Expr *CodeGen::IterateTableParallel(catalog::table_oid_t table_oid, ast::Expr *col_oids, ast::Expr *query_state,
+                                         ast::Expr *exec_ctx, ast::Identifier worker_name) {
+  ast::Expr *call = CallBuiltin(ast::Builtin::TableIterParallel, {Const32(table_oid.UnderlyingValue()), col_oids,
```

Comment:
This was done to de-duplicate some code in `IndexCreateTranslator`. `col_oids` can't be an `ast::Identifier` in that case since the col_oids comes from the QueryState.